carla-carma-integration/carla_carma_bridge/src/carla_carma_bridge/carla_to_carma_vehicle_status

```diff
@@ -81,10 +81,17 @@ def vehicle_status_callback(vehicle_status_msg):
     status.angle = vehicle_status_msg.control.steer * math.degrees(max_steering_angle)
     status.speed = vehicle_status_msg.velocity
 
+    # VehicleStatus from autoware_msgs has different parameter compare with autoware 1.12 and 1.14
     if vehicle_status_msg.control.reverse:
-        status.gearshift = Gear.REVERSE
+        try:
+            status.gearshift = Gear.REVERSE
```

Comment:
I'm not sure I understand what exception is being expected here. Can this assignment actually fail somehow? And if it does, it's not clear to me why setting a different field (which is not set in the normal case) would correct the error. Is this possibly handling two different message types with different fields or something?

Reply:
The autoware.ai version has been changed to 1.14.0 version, the VehicleStatus message content is different then before. ([CARMA 3.10.0 VehicleStatus message content](https://github.com/usdot-fhwa-stol/autoware.ai/blob/carma-system-3.10.0/messages/autoware_msgs/msg/VehicleStatus.msg#L10)) ([Previous version VehicleStatus message content](https://github.com/usdot-fhwa-stol/autoware.ai/blob/2f7218b6bf519ff49b563eb1794f5f3ede6f2481/messages/autoware_msgs/msg/VehicleStatus.msg#L10)). 
Adding the exception here could avoid the autoware message version mismatch between integration tool and CARMA